README.md

```
@@ -222,14 +222,14 @@ tools like `nvm`, `fnm`, etc...
 | [cline][cline]                       | `pnpm add -g cline`<br/> **OR** `npm i -g cline`<br/> **OR** See [Cline docs][cline-docs]                                                                                                      |
 | [goose][goose]                       | `brew install block-goose-cli`<br/> **OR** See [Goose docs][goose-docs]                                                                                                                        |
 
-> [!WARNING]  
+> [!WARNING]
 > These install commands are here for convenience, please always refer to the
 > official installation instructions from the respective ACP provider.
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)